GliaWidgets/Layout/NSLayoutConstraint.Extensions.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import UIKit
+
+extension Array where Element == NSLayoutConstraint {
+    func activate() {
+        NSLayoutConstraint.activate(self)
+    }
+
+    func deactivate() {
+        NSLayoutConstraint.deactivate(self)
+    }
+
+    // swiftlint:disable operator_whitespace
+    static func +=(
+        destination: inout [NSLayoutConstraint],
+        newConstraint: NSLayoutConstraint
+    ) {
+        // swiftlint:enable operator_whitespace
+        destination.append(newConstraint)
+    }
+
+    func constraints(with id: NSLayoutConstraint.Identifier) -> [NSLayoutConstraint] {
+        filter { $0.identifier == id.rawValue }
+    }
+}
+
+extension NSLayoutConstraint {
```

Comment:
Let's document this extension and it's types/methods.

Reply:
Added a few docs - d909ae32942ab85c0fe7851a61c36a533f017071
but perhaps I need to add just a comment with a description of the use-case for this function. What do you think, does it make sense?